vikunja/docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: vikunja_frontend_1
+      APP_PORT: 80
+
+  db:
+    image: mariadb:10@sha256:385c0ba6270307511a511ad915350736602c33fafb3f63f663c853920424c365
+    command: --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci
+    environment:
+      MYSQL_ROOT_PASSWORD: supersecret
+      MYSQL_USER: vikunja
+      MYSQL_PASSWORD: secret
+      MYSQL_DATABASE: vikunja
+    volumes:
+      - ${APP_DATA_DIR}/db:/var/lib/mysql
+    restart: unless-stopped
+  api:
+    image: vikunja/api:0.21@sha256:e541ef5599d08f50a24e8de1fd8136f9d0a7ef514916fac6d6bf8ac6c7c16712
+    environment:
+      VIKUNJA_DATABASE_HOST: db
+      VIKUNJA_DATABASE_PASSWORD: secret
+      VIKUNJA_DATABASE_TYPE: mysql
+      VIKUNJA_DATABASE_USER: vikunja
+      VIKUNJA_DATABASE_DATABASE: vikunja
+      VIKUNJA_SERVICE_JWTSECRET: <a super secure random secret>
+      DEVICE_DOMAIN_NAME: "${DEVICE_DOMAIN_NAME}"
+      VIKUNJA_SERVICE_FRONTENDURL: http://DEVICE_DOMAIN_NAME/
+    ports:
+      - 3456:3456
+    volumes:
+      - ${APP_DATA_DIR}/files:/app/vikunja/files
+    depends_on:
+      - db
+    restart: unless-stopped
+  frontend:
+    image: vikunja/frontend:0.21@sha256:fdd4582979beee4a6bef2d4d8e3744809b82ab2e198d708a829d7189a0ad9882
+    restart: unless-stopped
+    environment:
+      DEVICE_DOMAIN_NAME: "${DEVICE_DOMAIN_NAME}"
+      VIKUNJA_API_URL: http://DEVICE_DOMAIN_NAME:3456/api/v1
```

Comment:
You can update this from `VIKUNJA_API_URL: http://DEVICE_DOMAIN_NAME:3456/api/v1` to `VIKUNJA_API_URL: http://api:3456/api/v1`.